src/utils/getLowResUrl.ts

```diff
@@ -7,7 +7,7 @@ const size = (Math.ceil(CardSize) * PixelRatio.get()) / 5;
 
 export const getLowResUrl = (url: string) => {
   if (url?.startsWith?.(GOOGLE_USER_CONTENT_URL)) {
-    return `${url}?w=${size}`;
```

Comment:
So this was working previously by coincidence?

Reply:
@osdnk It was working because Google defaulted to 512px wide because it couldn't read this param. Documented here: https://github.com/rainbow-me/rainbow/pull/3007#discussion_r819601870